scripts/eip1559_deployer.ts

```diff
@@ -0,0 +1,457 @@
+import 'hardhat-deploy-ethers';
+import * as fs from 'fs';
+import mainnetData from '../mainnet_deployed.json';
```

Comment:
There is no need to include both, just the one for which we are deploying, see: https://github.com/ethersphere/storage-incentives/blob/master/scripts/deployer.ts#L54